provision.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+. ./semver.sh
```

Comment:
@kikito without reading the semver code below, semver is not applicable for 0.x versions, so does not apply to Kong